source/schema.txt

```diff
@@ -338,3 +338,424 @@ To delete a location filter from the map:
 .. |query-build-ref| replace:: :ref:`build-query`
 
 .. |query-build-type| replace:: :ref:`build-query`
+
+Export Your Schema
+------------------
+
+You can export your schema after analyzing it. This is useful for 
+sharing your schema and comparing schemas across collections.
+
+If you have not already done so, analyze your schema:
+
+1. Select your desired collection and click the :guilabel:`Schema` tab.
+
+#. Click :guilabel:`Analyze Schema`.
+
+.. note::
+   When |compass-short| analyzes your schema, it samples a random 
+   subset of documents from your collection. To learn more about 
+   sampling, see :ref:`Sampling <sampling>`.
+
+Once your schema has been analyzed, export your schema:
+
+1. In the top menu bar, click :guilabel:`Collection`.
+
+#. From the dropdown, click :guilabel:`Share Schema as JSON`.
+
+.. figure:: /images/compass/share-schema-json.png
+   :alt: Image showing Collection dropdown
+
+Your schema is copied to your clipboard as a |json| object.
+
+Schema Object Properties
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Schema objects have properties ``count`` and ``fields``:
+
+- ``count`` is an integer that represents the number of documents 
+  :ref:`sampled <sampling>` from the collection to generate the schema.
+
+- ``fields`` is an array of ``n`` metadata objects that represent each 
+  field in the documents analyzed during sampling. Each element in the 
+  ``fields`` array contains:
+
+.. list-table::
+  :header-rows: 1
+  :widths: 25 35 40
+
+  * - Field name
+    - Data type
+    - Description
+
+  * - ``name``
+    - String
+    - Name of ``fields[n]``, e.g. ``_id``.
+
+  * - ``path``
+    - String
+    - Path to ``fields[n]`` within a document.
+    
+  * - ``count``
+    - Integer
+    - The number of documents that ``fields[n]`` appears on.
+
+  * - ``types``
+    - Array
+    - List of ``n`` metadata objects that represent each data type that 
+      appears in ``fields[n]``.
+
+  * - ``types[n].name``
+    - String
+    - Data type of ``types[n]``.
+  
+  * - ``types[n].bsonType``
+    - String
+    - |bson| type of ``types[n]``.
+
+  * - ``types[n].path``
+    - String
+    - Path to ``name`` of ``fields[n]`` within a document.
```

Comment:
I think using `name` here is a little confusing because we have both `types[n].name` and `name`, so it's unclear which one we mean. What about: "Path to `fields[n]` where `fields[n]` has this data type."

Reply:
Went with: "Path to ``fields[n]`` within a document." which is consistent with the use of ``fields[n]`` as a stand-in for the document field, like _id. That also makes it the same as the description for ``path``, which is correct - it is always the same value as the parent's path.